test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTests.cs

```diff
@@ -1429,5 +1429,29 @@ public class ModelBindingTests
             Assert.Equal("test.txt", fileDetails.Filename);
             Assert.Equal("Test Content", fileDetails.Content);
         }
+
+        [Fact]
+        public async Task TryUpdateModel_DerivedTypeCasted_ReturnsDerivedAndBaseProperties()
```

Comment:
see other comment - I think this can be just `TryUpdateModel_ReturnDerivedAndBaseProperties`